fe/fe-core/src/main/java/com/starrocks/analysis/CreateMaterializedViewStmt.java

```diff
@@ -204,7 +204,8 @@ public void analyzeSelectClause(boolean ignoreCast) throws AnalysisException {
                 FunctionCallExpr functionCallExpr = (FunctionCallExpr) selectListItemExpr;
                 String functionName = functionCallExpr.getFnName().getFunction();
                 // current version not support count(distinct) function in creating materialized view
-                if (!isReplay && functionName.toLowerCase().equals("count") && functionCallExpr.isDistinct()) {
+                if (!isReplay && (functionName.equalsIgnoreCase("count")
+                        || functionName.equalsIgnoreCase("sum")) && functionCallExpr.isDistinct()) {
```

Comment:
Why not change it like this
```suggestion
                if (!isReplay && functionCallExpr.isDistinct()) {
```

Reply:
ok